riptide-nakadi/src/main/java/org/zalando/riptide/nakadi/Event.java

```diff
@@ -0,0 +1,72 @@
+package org.zalando.riptide.nakadi;
+
+/*
+ * ⁣​
+ * Riptide: Nakadi
+ * ⁣⁣
+ * Copyright (C) 2015 - 2016 Zalando SE
+ * ⁣⁣
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ​⁣
+ */
+
+import java.time.OffsetDateTime;
+import java.util.UUID;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import lombok.Value;
+
+@Value
+public class Event {
+
+    public enum DataOperation {
+
+        CREATE("C"),
+        UPDATE("U"),
+        DELETE("D"),
+        SNAPSHOT("S");
+
+        private final String value;
+
+        private DataOperation(final String value) {
+            this.value = value;
+        }
+
+        @JsonValue
+        public String getValue() {
+            return value;
+        }
+    }
+
+    @Value
+    public static class Metadata {
+
+        UUID eid;
+
+        OffsetDateTime occurredAt;
+
+        String flowId;
+
+    }
+
+    Metadata metadata;
+
+    String dataType;
+
+    DataOperation dataOp;
+
+    ObjectNode data;
```

Comment:
No, the support for datachange events should already be optional. The fields are available but must not be filled in by the stream.